Sources/Mustache/Transform.swift

```diff
@@ -48,7 +48,7 @@ public extension StringProtocol {
         case "uppercased":
             return uppercased()
         case "reversed":
-            return reversed()
+            return Substring(self.reversed())
```

Comment:
`reversed()` returns a `ReversedCollection` and messes the next transformations since It's no longer a `StringProtocol`.